### ISSUE TITLE
Will it work in ubuntu 19? (2020) I will use it for the first time that's why I am asking

### ISSUE THREAD

**Issue by 7Wdev** [skilbjo/exagear#2]

**Comment by skilbjo** [comment]
hey @7Wdev 

I believe eltechs runs natively on ubuntu/debian systems, so if that's what you're after, you shouldn't need my notes here to get it working on Arch Linux, which is what I was after. However, eltechs discontinued its product last year and its website (eltechs.com) is not up, so seems like the company has disappeared, so I'm surprised you even found this repo.

**Comment by 7Wdev** [comment]
> hey @7Wdev
> 
> I believe eltechs runs natively on ubuntu/debian systems, so if that's what you're after, you shouldn't need my notes here to get it working on Arch Linux, which is what I was after. However, eltechs discontinued its product last year and its website (eltechs.com) is not up, so seems like the company has disappeared, so I'm surprised you even found this repo.

Yea I noticed that, that's why I am wondering.
I mean those are the binaries why would they don't work anymore? Or maybe unbuntu 19 os instruction has been changed alot idk, however I am searching for someway to run amd64 app on armv64 for studying using...

**Comment by skilbjo** [comment]
Yeah good point. I think I am somewhat forgetting exactly what I all did here... I think I did this in 2016, and put this in a repo in 2017 and have since forgotten about it. But yeah if you have the binaries it should all work... I think the gist of what I've done here is just some minor tweaks to the `.deb` package to create a valid package for Arch Linux, but not modified anything else. (For context, Arch Linux uses `pacman` for installing packages, versus Debian/Ubuntu's `apt`... but you still can install `.deb` packages on Arch Linux by installing `dpkg`; but the details of the `.deb` package need to be modified slightly to get it working for Arch Linux, which is what this repo is all about). So if you have the `.deb` and the binaries, it should all work by default on Debian/Ubuntu...

**Comment by 7Wdev** [comment]
Aha, thank you
